### PR TITLE
Fix: Multi update

### PR DESCRIPTION
**Pull request by joannafawl**
#### Work done
- Use `[(ngModel)]` in form for select all checkbox instead of reactive form so that validation is updated on submit
- Reset selected staff in training service when pressing cancel on `add-training-details` page

#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: src/app/features/add-multiple-training/training-details/training-details.component.ts [hunk 1]
```diff
@@ -80,4 +80,9 @@ export class MultipleTrainingDetailsComponent extends AddEditTrainingDirective i
   private onError(error) {
     console.log(error);
```
Comment (Richard-Pentecost) [inline src/app/features/add-multiple-training/training-details/training-details.component.ts:81]: Should this onError method do anything more with this error other than logging it

**Review by Richard-Pentecost — APPROVED** [review]

File: src/app/features/add-multiple-training/training-details/training-details.component.ts (hunk 1 above)
Comment (joannafawl) [reply src/app/features/add-multiple-training/training-details/training-details.component.ts:81]: It's a bit of a weird one, in other parts of the code we either display server error messages or just console log the error but we don't have any server error messages. We could navigate to the `Sorry there is a problem with this service` page too if you think that's better?

Comment (Richard-Pentecost) [reply src/app/features/add-multiple-training/training-details/training-details.component.ts:81]: Yeah, I had something similar in one of my tickets. The issue with just console logging it is the user doesn't necessarily know that there has been a problem. Navigating to the error page maybe a good solution. Might be worth speaking to Aaron to see what he thinks

Comment (joannafawl) [reply src/app/features/add-multiple-training/training-details/training-details.component.ts:81]: I checked with Aaron and we automatically navigate to that page on error

**Pull request merged**